### PR TITLE
13 create analysis section with alignment and homology search tables

### PR DESCRIPTION
**Pull request by leticiamaiat**
Created homology search endpoint to get data by analysis ID and also it's analysis section that displays the results. 
Bases and Matchalignment are commented for further actions.

### CODE REVIEW

File: src/pages/HomologyAnalysisDetails.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        openSnackbar(true);
+    }
+    const onFailureGetAlignmentByIdAnalysis = (error) => {
+        showSnackbar(getMessage(error.errorDescription), 'error');
+        showLoader(false);
+    }
+
+    useEffect(() => {
+        
+        // /fetch(API_ROUTES.GET_TAXONOMY_BY_ID_ANALYSIS + '?idAnalysis=' + idAnalysis)
```
Comment (LuizMVB) [inline src/pages/HomologyAnalysisDetails.js:36]: Please, uncomment or remove this code

File: src/pages/HomologyAnalysisDetails.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    },
+    {
+        id: 'externalDatabaseId',
+        label: getMessage('olatcgHomologyTable.label.externalDatabaseId'  )
+    },
+    {
+        id: 'countryOrigin',
+        label: getMessage('olatcgHomologyTable.label.countryOrigin' )
+    },
+    // {
```
Comment (LuizMVB) [inline src/pages/HomologyAnalysisDetails.js:64]: Please, uncomment or remove this code

File: src/pages/HomologyAnalysisDetails.js (hunk 1 above)
Comment (leticiamaiat) [reply src/pages/HomologyAnalysisDetails.js:36]: Reason for comment:
Choose the best request method

File: src/pages/HomologyAnalysisDetails.js (hunk 2 above)
Comment (leticiamaiat) [reply src/pages/HomologyAnalysisDetails.js:64]: Reason for comment:
Commented ids require refinement due to returning large values

File: src/components/Slide.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import Slide from '@mui/material/Slide';
+
+
+
+
+const Transition = React.forwardRef(function Transition(props, ref) {                               
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+function AlertDialogSlide (props) {
```
Comment (LuizMVB) [inline src/components/Slide.js:17]: Change the component and file name to OlatcgAlertDialogSlide

**Review by LuizMVB — COMMENTED** [review]
Needs to resolve comments before merge

**Review by LuizMVB — APPROVED** [review]
Nice!

**Pull request merged**